src/main/java/com/lob/protocol/response/CheckResponse.java

```diff
@@ -137,6 +144,7 @@ public String toString() {
             ", trackingNumber='" + trackingNumber + '\'' +
             ", trackingEvents='" + trackingEvents + '\'' +
             ", expectedDeliveryDate=" + expectedDeliveryDate +
```

Comment:
I noticed that when adding `mail_type` to the resources. I believe the rule of thumb that was used is if it's a string, it's wrapped around single quotes. If it's a number of date, it's left as is. At any point, this is in a `toString` method that should never be used in production (just debugging). We can standardize everything in another PR.